sunscreen_runtime/src/builder.rs

```diff
@@ -398,119 +445,124 @@ mod linked {
             P: TryIntoPlaintext + TypeName,
         {
             let pt = self.plaintext_typed(message)?;
-            self.encrypt_symmetric_internal(Message::Plain(pt), private_key, None)
+            self.encrypt_symmetric_internal(Msg::from(pt), private_key, None)
         }
 
-        /// Encrypt a plaintext intended for sharing.
+        /// Encrypt a plaintext, adding the encryption statement to the proof and returning the
+        /// message to optionally be [encrypted again](`Self::encrypt_again`), that is, _shared_
```

Comment:
I am not sure how to update this comment but I think that "adding the encryption statement to the proof" is tripping me up, because I don't know whether that is the SDLP proof or the bullet proof (I know now after we chatted about it). It would reveal low level details to say "add the encryption statement to the SDLP proof", so I don't like that, but without more details on which proof we are talking about it is a tad confusing.

Reply:
I don't think it's too low level to say SDLP proof, elsewhere in the API docs we talk about the linked proof linking the SDLP and the ZKP together.

Reply:
Updated the comments